content/workshops/fullstack.md

```diff
@@ -0,0 +1,56 @@
+{
+    "type": "tutorial",
+    "layout": "type",
+    "title": "Fullstack Workshop",
+    "linktitle": "fullstack", 
+    "card_icon": "ti-layout",
+    "card_body": "Learn how to create and maintain a fullstack web application",
+    "weight": "1"
+}
+## Who is this workshop for?
+{{<break 1>}}
+This workshop is recommended to developers of all skill levels, that have basic knowledge of fullstack web-development, and are wanting to deploy applications on an AWS server. Cdev is an easy-to-use tool that is focussed on making serverless web-development easily attainable for all creators.  In this workshop we will teach you everthing you need to know to begin building complete web applications on your own.

```

Comment:
on and AWS Server -> on Aws

It can be unclear using 'server' there since we are deploying on a 'serverless' platform.